sw/device/silicon_creator/rom_ext/e2e/ownership/BUILD

```diff
@@ -101,7 +101,6 @@ ownership_transfer_test(
     fpga = {"changes_otp": True},
```

Comment:
I think nearly every ownership transfer test modifies the ownership state in flash such that we want to clear after the test.  Most of them also rely on installing a particular ownership configuration (by way of their choice of ROM_EXT) and clearing before the test is probably also a good idea.

Maybe we should update the `ownership_transfer_test` macro to make this behavior default and eliminate the `changes_otp` definition per test?